task_manager/tests/test_load_checkers/test_stateful_checker.py

```diff
@@ -169,3 +170,52 @@ async def test_requested_many_stateful_model():
     assert len(triggers) == 2
     assert isinstance(triggers[0], IncreaseTrigger)
     assert triggers[0].model == stub_stateful
+
+
+async def test_free_model_instance_after_some_time():
+    input_batch_queue = InputBatchQueue()
+    output_batch_queue = OutputBatchQueue()
+
+    request_info1 = dm.RequestInfo(
+        input=np.array(range(10)),
+        parameters={},
+    )
+    item1 = dm.MinimalBatchObject(
+        uid="1",
+        requests_info=[request_info1],
+        model=stub_stateful,
+        status=dm.Status.CREATED,
+        source_id="new_source_id",
+    )
+    await input_batch_queue.put(item1)
+
+    receiver_streams_combiner = ReceiverStreamsCombiner(output_batch_queue)
+    model_instances_storage = ModelInstancesStorage(receiver_streams_combiner)
+    bs = BaseSender()
+    bs.last_sent_batch = (
+        time.time() - config.load_analyzer.stateful_checker.keep_model - 1
+    )
+    model_instance = dm.ModelInstance(
+        stub_stateful,
+        "previous_source_id",
+        bs,
+        BaseReceiver(),
+        lock=False,
+        hostname="test",
+        running=True,
+        name="test",
+        num_gpu=None,
+        current_processing_batch=None,
+    )
+    model_instances_storage.add_model_instance(model_instance)
+    checker = StatefulChecker(
+        model_instances_storage, input_batch_queue, output_batch_queue, config=config
+    )
+    triggers = checker.make_triggers()
+    assert len(triggers) == 1
+    assert isinstance(triggers[0], IncreaseTrigger)
```

Comment:
The test is about releasing the model, but you use `IncreaseTrigger` instead of `DecreaseTrigger`